app/controllers/investigations/record_emails_controller.rb

```diff
@@ -18,7 +18,7 @@ def create
 
     if @email_correspondence_form.valid?
 
-      result = AddEmailToCase.call(
+      AddEmailToCase.call(
```

Comment:
```suggestion
      AddEmailToCase.call!(
```
Throws exception on failure, otherwise we will not know if it failed and may erroneously report to the user that it succeeded.